sql/moz-fx-data-marketing-prod/ga_derived/mobile_installs_adjust_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+description: Aggregated metrics from Adjust data
+friendly_name: Adjust mobile installs
+labels:
+  incremental: true
+  schedule: daily
+owners:
+  - rbaffourawuah@mozilla.com
+scheduling:
+  dag_name: bqetl_google_analytics_derived
```

Comment:
do we really want to put all the Adjust ETLs under Google Analytics?

Reply:
I am not sure about this and so I am open to recommendations from you and Sean.